### PR TITLE
[Merged by Bors] - chore: rename Finset.card_doubleton to Finset.card_pair

### PR DESCRIPTION
**Pull request by jcommelin**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/Finset/Card.lean [hunk 1]
```diff
@@ -129,9 +129,11 @@ theorem card_insert_eq_ite : card (insert a s) = if a ∈ s then s.card else s.c
 #align finset.card_insert_eq_ite Finset.card_insert_eq_ite
 
 @[simp]
-theorem card_doubleton (h : a ≠ b) : ({a, b} : Finset α).card = 2 := by
+theorem card_pair (h : a ≠ b) : ({a, b} : Finset α).card = 2 := by
   rw [card_insert_of_not_mem (not_mem_singleton.2 h), card_singleton]
-#align finset.card_doubleton Finset.card_doubleton
+#align finset.card_doubleton Finset.card_pair
+
+@[deprecated] alias Finset.card_doubleton := Finset.card_pair
```
Comment (jcommelin) [inline Mathlib/Data/Finset/Card.lean:136]:
```suggestion
@[deprecated] alias card_doubleton := Finset.card_pair
```

**Review by YaelDillies — APPROVED** [review]
Thanks!

maintainer merge

**Comment by YaelDillies** [comment]
maintainer merge

**Comment by urkud** [comment]
Thanks! :tada:
bors merge

**Pull request closed without merging**